lib/Dialect/TritonGPU/Transforms/F32DotTC.cpp

```diff
@@ -45,6 +45,15 @@ class TF32x3 : public OpRewritePattern<DotOp> {
                                           ArrayRef<Value>{value})
           .getResult()[0];
     };
+    auto zero_like = [&](Value c) -> Value {
```

Comment:
nit: `zeroLike` to follow the convention

Reply:
Changed.